### PR TITLE
Project submitted

### PR DESCRIPTION
**Pull request by LKWLaLa**

### CODE REVIEW

File: bin/tictactoe [hunk 1]
```diff
@@ -1,3 +1,7 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+
+
+Game.start
```
Comment (randallreedjr) [inline bin/tictactoe]: Great bin file, short length is ideal. However, we would prefer to see the CLI logic separated from the game logic in its own class (e.g. TicTacToeCLI). Splitting the interface from the implementation of the game allows for multiple interfaces (e.g. a web app) without requiring changes to the game code.

File: lib/game.rb [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+  def initialize(player_1 = Human.new("X"),player_2 = Human.new("O"),board = Board.new)
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+  end
+
+  def current_player
+    counter = self.board.turn_count
+    if counter % 2 == 0
```
Comment (randallreedjr) [inline lib/game.rb]: Use [`Integer#even?`](http://ruby-doc.org/core-1.8.7/Integer.html#method-i-even-3F) or [`Integer#odd?`](http://ruby-doc.org/core-1.8.7/Integer.html#method-i-odd-3F)

`@board.turn_count.even? ? @player_1 : @player_2`

File: lib/game.rb [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+    end
+  end
+
+  def over?
+    self.draw? ||  self.won? 
+  end
+
+      
+  def draw?
+   self.board.full? && self.won? == nil
```
Comment (randallreedjr) [inline lib/game.rb]: Avoid equality comparison with `nil`. Instead, you can use the [`.nil?`](http://ruby-doc.org/core-2.2.0/NilClass.html#method-i-nil-3F) method or simply `!self.won?`.

File: lib/game.rb [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+
+  def won?
+      WIN_COMBINATIONS.detect do |line|
+        line.all?{|i| self.board.cells[i] == "X" } || line.all?{|i| self.board.cells[i] == "O"}
+      end
+    end
+
+  def winner
+    if won?
+      line = won?
```
Comment (randallreedjr) [inline lib/game.rb]: You can accomplish both of these steps using if-assignment.
```
if line = won?
  self.board.cells[line[0]]
end
```
If `won?` returns nil, the expression `line = won?` will also return nil and the body of the if statement will not be executed.

File: lib/players/human.rb [hunk 5]
```diff
@@ -0,0 +1,8 @@
+class Human < Player
+
+  def move(board)
+    puts "Your turn #{self.token}! Choose a cell number (1-9):"
+    input = gets.strip
```
Comment (randallreedjr) [inline lib/players/human.rb]: You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board)
  puts "Your turn #{self.token}! Choose a cell number (1-9):"
  gets.strip
end
```

**Comment by randallreedjr** [comment]
@LKWLaLa Great job! I left you a few comments, feel free to respond. Closing this PR.

**Comment by LKWLaLa** [comment]
@randallreedjr  Thank you so much for the feedback!  Will go over these notes.  Much appreciated!

**Pull request closed without merging**